cql2pgjson/src/test/java/org/z3950/zing/cql/cql2pgjson/ForeignKeyGenerationIT.java

```diff
@@ -50,6 +51,11 @@ public void foreignKeySearch0() throws Exception {
     assertThat(cql("tableb.prefix == x0"), is(empty()));
   }
 
+  @Test
+  public void foreignKeySearchParentChild() throws Exception {
+    assertThat(cql("tabled.prefix == a1"), containsInAnyOrder("test0", "test1"));
```

Comment:
You may want to rename cql(String) to cqla(String) and create a new cqlb(String):

    private List<String> cqlb(String cql) {
      try {
        String sql = cql2pgJson.toSql(cql).toString();
        return firstColumn("select jsonb->>'name' from tableb " + sql);
      } catch (QueryValidationException e) {
        throw new RuntimeException(e);
      }
    }

Then you can query the other direction using tableb as the current table and use tableb.* in CQL.